style.css

```diff
@@ -0,0 +1,15 @@
+*{
+  background-color: black;
+  color: white;
+  text-align: center;
+}
+header{
+    width: 764px;
+    height: 246px;
+    margin: auto;
+    background-image: url(bus.jpg);
+}
+img{
+  width: 250px;
```

Comment:
image heights differ among images.  Consider editing the images to preserve aspect ration while having a uniform height